src/execute.ts

```diff
@@ -12,27 +19,21 @@ export async function executePiper (
     ? flags.concat(JSON.parse(process.env.defaultsFlags))
     : flags
 
-  const piperPath = internalActionVariables.piperBinPath
-  const containerID = internalActionVariables.dockerContainerID
-  let piperError = ''
-  let options = {
+  const handleFatalLog = (data: string): void => { data.includes('fatal') ? setFailed(data) : info(data) }
+
+  let options: ExecOptions = {
```

Comment:
As a general best practice to keep variable declaration/initialization before the place they are actually used, can we move it down, to line ~52?